pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -348,11 +348,11 @@ var DefaultLaggingRangesThreshold = 3 * time.Minute
 // DefaultLaggingRangesPollingInterval is the default polling rate at which
 // lagging ranges are checked and metrics are updated.
 var DefaultLaggingRangesPollingInterval = 1 * time.Minute
-
 var Quantize = settings.RegisterDurationSettingWithExplicitUnit(
 	settings.ApplicationLevel,
 	"changefeed.resolved_timestamp.granularity",
 	"the granularity at which changefeed progress are quantized to make tracking more efficient",
-	0,
+	time.Duration(metamorphic.ConstantWithTestRange("changefeed."+
+		"resolved_timestamp.granularity", 0, 0, 20))*time.Second,
```

Comment:
I was originally going to make the max higher, but ran into some flakiness with larger durations and tests like TestChangefeedLaggingSpanCheckpointing that depend on spans diverging within the SucceedsSoon duration.

Reply:
I wonder if we should also make it metamorphic for roachtests since their changefeeds will run longer and consequently are more likely to see an impact. You'd also probably be able to set a higher upper bound there.

Note that it's a more involved code change, see https://cockroachlabs.slack.com/archives/C023S0V4YEB/p1733846452422519?thread_ts=1733846019.532919&cid=C023S0V4YEB